### PR TITLE
[AN-5908] First time UX for team admins

### PR DESCRIPTION
**Pull request by JPonte**
#### APK
[Download build #10082](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10082/artifact/build/artifact/wire-dev-PR1378-10082.apk)
[Download build #10083](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10083/artifact/build/artifact/wire-dev-PR1378-10083.apk)
[Download build #10087](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10087/artifact/build/artifact/wire-dev-PR1378-10087.apk)
[Download build #10089](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10089/artifact/build/artifact/wire-dev-PR1378-10089.apk)
[Download build #10090](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10090/artifact/build/artifact/wire-dev-PR1378-10090.apk)
[Download build #10092](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10092/artifact/build/artifact/wire-dev-PR1378-10092.apk)
[Download build #10099](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10099/artifact/build/artifact/wire-dev-PR1378-10099.apk)
[Download build #10100](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10100/artifact/build/artifact/wire-dev-PR1378-10100.apk)
[Download build #10102](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10102/artifact/build/artifact/wire-dev-PR1378-10102.apk)
[Download build #10105](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10105/artifact/build/artifact/wire-dev-PR1378-10105.apk)
[Download build #10108](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10108/artifact/build/artifact/wire-dev-PR1378-10108.apk)
[Download build #10110](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10110/artifact/build/artifact/wire-dev-PR1378-10110.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Land on first-time UI. Send a connection request to anybody. Now in the conversations list you see the outgoing request, but the contacts icon is still in the center

**Comment by someoneEsle** [comment]
- [ ] Bug 2
Emptying the archive makes the contacts icon move to the center

**Comment by someoneEsle** [comment]
- [ ] Bug 3
The owner of the team should keep seeing the invite button in the Search UI for as long as there are no other members in the team -- regardless of whether or not they're connected to guests. Right now connecting to a guest will dismiss the invite button in the search UI, which is not the intended behavior.

**Comment by someoneEsle** [comment]
Looks good, can you give @eikedrescher a phone with this build? He wants to double check

**Comment by JPonte** [comment]
build again

File: app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala [hunk 1]
```diff
@@ -249,10 +254,24 @@ class NormalConversationFragment extends ConversationListFragment {
     subs += topToolbar.onRightButtonClick(_ => getActivity.startActivityForResult(PreferencesActivity.getDefaultIntent(getContext), PreferencesActivity.SwitchAccountCode))
 
     listActionsView.setCallback(listActionsCallback)
+    listActionsView.setScrolledToBottom(!conversationListView.canScrollVertically(1))
+
+    noConvsMessage.setOnClickListener(new OnClickListener {
+      override def onClick(v: View): Unit =
+        getControllerFactory.getPickUserController.showPickUser(IPickUserController.Destination.CONVERSATION_LIST)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala]: please use `inject[IPickUserController]` instead - in face I forgot to remove `BaseFragment` from this class.

File: app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala [hunk 2]
```diff
@@ -249,10 +254,24 @@ class NormalConversationFragment extends ConversationListFragment {
     subs += topToolbar.onRightButtonClick(_ => getActivity.startActivityForResult(PreferencesActivity.getDefaultIntent(getContext), PreferencesActivity.SwitchAccountCode))
 
     listActionsView.setCallback(listActionsCallback)
+    listActionsView.setScrolledToBottom(!conversationListView.canScrollVertically(1))
+
+    noConvsMessage.setOnClickListener(new OnClickListener {
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala]: `RichView#onClick()`

File: app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala [hunk 3]
```diff
@@ -190,34 +192,37 @@ class NormalConversationFragment extends ConversationListFragment {
...
   }
 
   lazy val loadingListView = view[View](R.id.conversation_list_loading_indicator)
-  lazy val listActionsView = returning(view[ListActionsView](R.id.lav__conversation_list_actions)) { v =>
-    archiveEnabled.onUi(v.setArchiveEnabled)
+  lazy val listActionsView = returning(view[ListActionsView](R.id.lav__conversation_list_actions)){ v =>
+   archiveEnabled.onUi(v.setArchiveEnabled(_))
+    hasConversationsAndArchive.onUi{
+      case (false, false) => v.setContactsCentered(true)
+      case _ => v.setContactsCentered(false)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/conversationlist/ConversationListFragment.scala]: map to boolean and call `v.setContactsCentered(value)` just once

File: app/src/main/java/com/waz/zclient/paintcode/WireDrawable.scala [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.waz.zclient.paintcode
+
+import android.graphics.drawable.Drawable
+import android.graphics.{ColorFilter, Paint}
+
+trait WireDrawable extends Drawable {
```
Comment (deanrobertcook) [inline app/src/main/java/com/waz/zclient/paintcode/WireDrawable.scala]: What's the point of this class exactly?

File: wire-core/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1271,4 +1271,7 @@
 
     <string name="app_entry_email_sent">Email sent</string>
     <string name="teams_set_username_subtitle">Usernames help people find you.</string>
+
+    <string name="empty_list_onboarding">Start a conversation or create a group.</string>
+    <string name="pick_user_teams_invite">Invite people to join the team</string>
```
Comment (deanrobertcook) [inline wire-core/src/main/res/values/strings.xml]: We don't have these strings anywhere else?

File: app/src/main/java/com/waz/zclient/paintcode/WireDrawable.scala (hunk 4 above)
Comment (JPonte) [reply app/src/main/java/com/waz/zclient/paintcode/WireDrawable.scala]: To use with the drawables from the new Kit thing. So far it's just one, but we might need to use it for more if we keep using that.

**Pull request merged**